src/weather_gen.cpp

```diff
@@ -265,7 +265,8 @@ void weather_generator::test_weather() const
     const time_point begin = calendar::turn;
     const time_point end = begin + 2 * calendar::year_length();
     for( time_point i = begin; i < end; i += 200_turns ) {
-        w_point w = get_weather( tripoint_zero, to_turn<int>( i ), 1000 );
+        //@todo: a new random value for each call to get_weather? Is this really intended?
+        w_point w = get_weather( tripoint_zero, to_turn<int>( i ), rng_bits() );
```

Comment:
Looks like this might be a merge resolution mistake?

Reply:
Yes it was, and I've cleaned it up.